addon/components/frost-bunsen-cell/component.js

```diff
@@ -1,12 +1,11 @@
 import _ from 'lodash'
-import Ember from 'ember'
 import computed, {readOnly} from 'ember-computed-decorators'
 
-import PropTypesMixin, {PropTypes} from 'ember-frost-bunsen/mixins/prop-types'
+import FrostComponent, {PropTypes} from 'ember-frost-component'
```

Comment:
Well the alternative is to figure out how to bake this logic directly into `Ember.Component` but then I think we run the risk of people assuming `propTypes` and `getDefaultPropTypes` exist within Ember itself. In that scenario if an addon uses these concepts and for some reason the consumer is missing `ember-frost-component` or whatever we decide to call it then there components will silently fail to validate `propTypes` or apply the defaults in `getDefaultProps()`. So with that being said I think the approach I took is the cleaner way.